test/functional/change_stream_tests.js

```diff
@@ -1522,14 +1506,322 @@ describe.skip('Change Streams', function() {
         assert.ifError(err);
 
         var database = client.db('integration_tests');
-        var changeStreamTest = database.collection('standAloneTest').watch();
-        changeStreamTest.hasNext(function(err, result) {
-          assert.equal(null, result);
-          assert.ok(err);
-          assert.equal(err.message, 'The $changeStream stage is only supported on replica sets');
+        const collection = database.collection('standAloneTest');
+
+        collection.insertOne({ d: 1 }).then(() => {
+          var changeStreamTest = collection.watch();
+          changeStreamTest.hasNext(function(err, result) {
+            assert.equal(null, result);
+            assert.ok(err);
+            assert.equal(err.message, 'The $changeStream stage is only supported on replica sets');
+
+            changeStreamTest.close(() => client.close(done));
+          });
+        });
+      });
+    }
+  });
+
+  it('Should include a default startAtClusterTime field when starting a fresh change stream', {
```

Comment:
I think this is probably a very nice opportunity to try to come up with JSON/YAML tests for change streams, what do you think? This would really help everyone implementing the new features in a standardized way, and would make it very easy for others to add tests. TBH if you came up with a format and moved a _single_ test into it, I bet others would fill in all the rest of the cases